babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    presets: [
+        [
+            '@babel/preset-env',
+            {
+                targets: {
+                    node: 'current',
```

Comment:
@jairo-bc since it's used in browsers we should probably target the same browsers (not node) as cornerstone does: 
https://github.com/bigcommerce/cornerstone/blob/master/webpack.common.js#L33

Reply:
Good point. Added that